### PR TITLE
fixed **crowdstrike-submit-sample** on v2

### PR DESCRIPTION
**Pull request by jochman**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/20497

## Description
multipart was sent two times, now it's sending the file in the right api convention.

## Screenshots
image

## Does it break backward compatibility?
   - No

## Must have
- [X] Tests
- [ ] Code Review

## Technical writer review
- [ ] [CHANGELOG](https://github.com/demisto/content/blob/fix-falcon-sandbox-multipart/Integrations/integration-CrowdStrikeFalconSandbox_CHANGELOG.md)

### CODE REVIEW

**Comment by Itay4** [comment]
lets test this with both v1 and v2

File: Integrations/integration-CrowdStrikeFalconSandbox.yml [hunk 1]
```diff
@@ -245,11 +245,16 @@ script:
...
 
-    function submitFile (entryId, environmentId) {
+    function submitFile(entryId, environmentId) {
         var requestUrl = version === 'v1' ? serverUrl + '/api/submit' : serverUrl + '/api/v2/submit/file';
+        var multipart_content = null;
         if (version === 'v2') {
             HEADERS['Content-Type'] = ['application/x-www-form-urlencoded'];
+            multipart_content = {environment_id: environmentId}; // For API v1
+        } else {
+            multipart_content = {environmentId: environmentId}; // For API v2
```
Comment (yardensade) [inline Integrations/integration-CrowdStrikeFalconSandbox.yml]: You change the logic to be opposite

**Comment by jochman** [comment]
> lets test this with both v1 and v2

we have no instance of v1 and the account can't be configured to v1.

**Comment by kirbles19** [comment]
@jochman Done.

**Comment by yardensade** [comment]
Build seem to fail, let's make sure our playbooks covers the issue well

**Pull request merged**